internal/scheduler/schedulerapp.go

```diff
@@ -203,3 +207,15 @@ func createLeaderController(config LeaderConfig) (LeaderController, error) {
 		return nil, errors.Errorf("%s is not a value leader mode", config.Mode)
 	}
 }
+
+func loadConfig() (*rest.Config, error) {
```

Comment:
I'd be tempted to call this `loadClusterConfig` as there's lots of other config floating around in this package.